integration_test/src/test_runner/test_generator_util.rs

```diff
@@ -0,0 +1,168 @@
+use crate::test_runner::{
+    fetch_row, get_column_attribute, get_column_count, get_data, Error, Result, TestEntry,
+};
+use lazy_static::lazy_static;
+use odbc::{safe::AutocommitOn, Allocated, NoResult, Statement};
+use odbc_sys::{CDataType, Desc, SqlDataType, USmallInt};
+use serde_json::{Number, Value};
+use std::string::ToString;
+
+const GENERATED_TEST_DIR: &str = "../resources/generated_test";
```

Comment:
```suggestion
const GENERATED_TEST_DIR: &str = "./resources/generated_test";
```
Same comment here about the `resources` directory location being in same directory as opposed to parent.